python/lsst/ts/salobj/configurable_csc.py

```diff
@@ -272,11 +269,12 @@ def _make_config_label_dict(self):
         The result should be put in self.config_label_dict
         """
         labels_path = self.config_dir / "_labels.yaml"
-        if not labels_path.is_file():
-            self.log.warning(f"{labels_path} not found")
+        try:
+            with open(labels_path, "r") as f:
+                labels_raw = f.read()
+        except OSError as e:
+            self.log.warning(f"{labels_path} not found: {e}")
```

Comment:
"not found" seems like the wrong message, since there might be other errors. Please consider something like:
```
f"Could not read labels {labels_path}: {e}"
```